helm/charts/amplication/values-staging-os.yaml

```diff
@@ -23,11 +23,14 @@ amplication-server:
       HOST: "https://server.staging-os.amplication-dev.com"
```

Comment:
I am unsure if we need the '-dev' here @mshidlov. You asked to compare this commit to the last one in this pull request, and it is with the '-dev' here and in the other commits.